app/models/hathi_trust_links.rb

```diff
@@ -0,0 +1,72 @@
+class HathiTrustLinks
+  attr_reader :document
+  def initialize(document)
+    @document = document
+  end
+
+  def present?
+    (hathitrust_work_id.present? || hathitrust_item_id.present?) && !fulltext_available?
+  end
+
+  # According to https://www.hathitrust.org/hathifiles_description access will be "allow"
+  # even in the case that the item is copyrighted in the US, or only Public Domain in the US
+  # (and therefore not available to users outside the US).
+  def publicly_available?
```

Comment:
We could flip this to fit more of the Stanford domain to be `stanford_only?` (as we basically reverse this when we check it elsewhere).  It was just a smaller change to keep the previous name.

Reply:
👍 ; I think `publicly_available?` probably makes sense in this context (and once ETAS is over, the stuff that's SU only won't be available)